css/styles.css

```diff
@@ -468,3 +498,257 @@ header {
+    background-position: right top;
+    background-size: contain;
+    flex-direction: row;
+    justify-content: flex-start;
+    height: 798px;
+  }
+
+  .headline .icons-container {
+    order: 0;
+    width: auto;
+    height: auto;
+    margin-left: 28px;
+    flex-direction: column;
+    align-self: center;
+  }
+
+  .headline-text {
+    order: 1;
+    margin-left: 75px;
+    align-self: center;
+    width: 38%;
+  }
+
+  .headline h1 {
+    margin: 0 0 14px;
+    font-size: 72px;
+    line-height: 81.5px;
+  }
+
+  .headline h2 {
+    margin: 0 0 14px;
+    font-size: 32px;
+    font-weight: 500;
+  }
+
+  .headline p {
+    margin: 0;
+    font-size: 20px;
+    line-height: 32px;
+  }
+
+  .works {
+    padding: 135px 142px 130px;
+  }
+
+  .works > h1 {
+    border: 0;
+    justify-self: flex-start;
+    margin: 0 0 114px;
+    padding: 0 12px 0;
+    box-sizing: border-box;
+    width: 100%;
+    display: flex;
+    white-space: nowrap;
+  }
+
+  .works-line {
+    border-bottom: 1px solid #212121;
+    width: 100%;
+    align-self: center;
+    margin-left: 24px;
+  }
+
+  .featured-work {
+    margin-top: 0;
+    flex-direction: row;
+  }
+
+  .featured-img {
+    flex: 0 1 60%;
+  }
+
+  .featured-work-right-container {
+    padding-left: 24px;
+    flex: 0 1 40%;
+  }
+
+  .featured-work h2 {
+    margin: 0;
+    font-size: 40px;
+  }
+
+  .featured-work p {
+    font-size: 20px;
+    line-height: 28px;
+    margin-bottom: 24px;
+  }
+
+  .featured-work .button {
+    margin-top: 24px;
+  }
+
+  .card-works {
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+    background-image: linear-gradient(180.45deg, rgba(38, 38, 38, 0) 0.75%, rgba(38, 38, 38, 0.9) 84.18%), url(card-background-desktop.png);
+    background-repeat: no-repeat;
+    background-size: contain;
+    height: 435px;
+    width: 369px;
+  }
+
+  .card-works p {
+    font-size: 17px;
+    margin-top: 24px;
+    margin-bottom: 24px;
+  }
+
+  .card-works .tags {
+    margin-bottom: 16px;
+  }
+
+  .card-works .button {
+    display: none;
+    margin: 0;
+    padding: 17px;
+  }
+
+  .about p {
+    margin-left: 142px;
+    font-size: 20px;
+    line-height: 32px;
+    height: auto;
+    width: 40%;
+  }
+
+  .about .button {
+    margin-left: 142px;
+    margin-bottom: 139px;
+  }
+
+  .about-list li ul {
+    margin-bottom: 0;
+  }
+
+  .card-works:hover {
+    background-image: url(card-background-desktop.png);
+    border: 2px solid #091e42;
+    background-origin: border-box;
+    flex-direction: column-reverse;
+    justify-content: flex-start;
+  }
+
+  .card-works:hover h2,
+  .card-works:hover p,
+  .card-works:hover ul {
+    display: none;
+  }
+
+  .card-works:hover .button {
+    display: initial;
+  }
+
+  .about {
+    background-image: url(illustration-about-desktop.png);
+    background-repeat: no-repeat;
+    background-position: right 99px;
+    background-size: 45%;
+  }
```

Comment:
- [ ] [OPTIONAL] please note, this background image on wide-screens breaks, I figured out your screen might not be as big so I tested for you the breakpoint is `1600px` wide If you fix that your portfolio would look fantastic, I think most users now have 1920px screen-wide I just wanted you to have that edge of advantage :) 
![image](https://user-images.githubusercontent.com/87689547/149230353-5d38740c-f7da-4273-a54d-302527df52b5.png)

Reply:
Thank you, it's fixed now I think